vita3k/config/src/config.cpp

```diff
@@ -97,6 +97,10 @@ ExitCode serialize_config(Config &cfg, const fs::path &output_path) {
     if (!fs::exists(output.parent_path()))
         fs::create_directories(output.parent_path());
 
+    std::string fuck = output_path.string();
+
+    LOG_WARN("YAML OUTPUT PATH: {}", fuck);
```

Comment:
??? WTF ?

Reply:
Sorry.

That was a debug line I was using that I thought I had removed prior to committing. My bad.

It's been removed in the latest push.